### ISSUE TITLE
Rasberry Pi Wifi Upload Script

### ISSUE THREAD

**Issue by Squidtoon99** [DallasFormulaRacing/PowertrainLib#4]
We need a way to move the data into the cloud from the Pi after testing sessions. I want an autonomous solution that will upload code as needed.

## Implementation

Write a python script that will run on startup that scans for Wi-Fi networks. If it finds the workshop network: It will connect to the network and upload all new files to https://lambda.dallasformularacing.com

Send a discord webhook message to the channel to notify people of data upload completion.

Ensure that the Wi-Fi network is switched back to the car ethernet after data upload is completed.

## Tasks
- [ ] Use the sub processes python module to detect a valid WIFI network, preferably the one in the shop
- [ ] Set up mock data transferring script to send data to the lambda endpoint when the correct WIFI network has been found
- [ ] Read over the TicketMaker in IC-Repo and create a similar discord bot
- [ ] create a new channel in the FSAE discord to send the message to

**Comment by GHAFHA** [comment]
Box API is completed waiting on permission from enterprise admin before continuing

**Comment by Squidtoon99** [comment]
Needs to be merged into main repo after #10 and #11
